osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/src/main/java/org/opensmartgridplatform/simulator/protocol/iec60870/server/Iec60870ASduHandlerMap.java

```diff
@@ -0,0 +1,22 @@
+package org.opensmartgridplatform.simulator.protocol.iec60870.server;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openmuc.j60870.TypeId;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Iec60870ASduHandlerMap {
```

Comment:
Main function of the class is to act as a Map for the ASdu Handlers, Furthermore I would not like using plural in class names. 

It is currently used for "controlling station" => "controlled station" only, but I'll guess it could be used in any situation where ASdus are being received (as long as either handling an ASdu with a specific type id behaves the same for both communication types or different ASdu types are used for each communication type)